src/components/BulkEditPane/BulkEditListResult/BulkEditInApp/ContentUpdatesForm/helpers.js

```diff
@@ -1,5 +1,5 @@
-import moment from 'moment/moment';
 import uniqueId from 'lodash/uniqueId';
+import dayjs from '../../../../../utils/dayjs';
```

Comment:
How i understood we need to use `dayjs` from `stripes` (example below)
`import { dayjs } from '@folio/stripes/components';`
CC: @zburke Could you please correct me if i am wrong?

Reply:
FYI, @Dmitriy-Litvinenko, @vashjs, I updated the parent-epic, [STRIPES-967](https://folio-org.atlassian.net/browse/STRIPES-967), to include this import example and be explicit about this expectation.